.github/workflows/build.yml

```diff
@@ -4,7 +4,6 @@ jobs:
   build:
     strategy:
       matrix:
-        go-version: [~1.17, ^1]
```

Comment:
Should we just use `charmbracelet/meta/.github/workflows/build.yml@main`?

Reply:
there are some custom envs set, and afaik `uses` doesn't allow to set env :/

Reply:
> there are some custom envs set, and afaik `uses` doesn't allow to set env :/

Ok